modules/installing-rhv-setting-up-ca-certificate.adoc

```diff
@@ -14,17 +14,23 @@ Later, you provide the certificate to the installation program.
 
 .Procedure
 
-. Download the CA certificate using either of these methods:
-** Go to the Manager's webpage and click the *CA Certificate* link. The URL of the Manager's webpage uses this format: `https://<engine-fqdn>/ovirt-engine/`.
+. Use either of these two methods to download the CA certificate:
+** Go to the Manager's webpage, `https://<engine-fqdn>/ovirt-engine/`. Then, under *Downloads*, click the *CA Certificate* link.
 ** Run the following command:
 +
 ----
 $ curl -k 'https://<engine-fqdn>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA' -o /tmp/ca.pem  <1>
 ----
 <1> For `<engine-fqdn>`, specify the fully qualified domain name of the {rh-virtualization} Manager, such as `rhv-env.virtlab.example.com`.
 
+. Optional: Configure the CA file to grant rootless user access to the Manager. Set the CA file permissions to have an octal value of `0644`
+and a symbolic value of `-rw-r--r--`. For example, in Linux, run:
++
+----
+$ sudo chmod 0644 /tmp/ca.pem
```

Comment:
Just a nit note:
- after copying the file from /tmp/ca.pem to /etc/pki/ca-trust/source/anchors/ENGINE_FQDN.pem
- $ sudo chmod 0644 /tmp/ca.pem

Reply:
Thanks. Updated. Please have another look.